### ISSUE TITLE
Loss of sx ox eternal hard drive support

### ISSUE THREAD

**Issue by ranshiro** [cathery/sys-con#108]
after using sys-con sx os will no longer read an attached usb drive. also my 8bitdo wireless adaptor stopped working. 

is there a way to deactivate sys-con? i removed the files from my sdcard but the switch still won't recognize anything that is plugged into a usb port.

**Comment by cathery** [comment]
Remove the folder located in `.../titles/690000000000000D`. Otherwise I can't really help you

**Comment by ranshiro** [comment]
I actually never got sys-con to work on switch, even though other people with sx os got it work. I removed everything else from my titles folder and my config folder except for the sys-con files but still no such luck. the only thing it did do is everything that was plugged into the switch stopped working. and then even removing the sys-con files from titles and config didn't change anything.

Eventually after a lot of trial and error and other shenanigans, I got the usb drive to be read again and I got the 8bitdo to work by following what other people said on here to upgrade its own firmware. 

Anyway, definitely no complaints. I know that you are focused on Atmosphere and I took a risk. I still appreciate your hard work, and maybe one day I'll figure out what the issue is for me and get it all working. 

Cheers
